src/cmds/create.js

```diff
@@ -81,9 +75,8 @@ module.exports = () => {
       console.log(`You can start coding by opening ./${directoryNameFor(answers.name)}/index.ts`)
       console.log(`Visit https://github.com/livestorm/livestorm-plugin for documentation`)
     } catch(err) {
-      console.log(err)
+      rimraf.sync(pathForPlugin(answers.name))
```

Comment:
Ca peut throwe une exception qu'on a peut-être besoin de catcher ou pas ?

Reply:
Pas grave de toute façon c'est juste pour du clean up histoire de pas laisser des dossiers à moitié créés :smile:

Reply:
I changed this line for the async version of it, that will at least allow finishing the error handling workflow